new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMapOverride.java

```diff
@@ -0,0 +1,679 @@
+        .append(" }]");
+    logger.log(Level.INFO, "ConfigMap Patch String: {0}", patchStr);
+
+    patch = new V1Patch(new String(patchStr));
+    boolean cmPatched = assertDoesNotThrow(() ->
+            patchDomainCustomResource(domainUid, domainNamespace, patch, "application/json-patch+json"),
+        "patchDomainCustomResource(configMap)  failed ");
+    assertTrue(cmPatched, "patchDomainCustomResource(configMap) failed");
+
+    patchStr = new StringBuffer("[{");
+    patchStr.append(" \"op\": \"replace\",")
+        .append(" \"path\": \"/spec/restartVersion\",")
+        .append(" \"value\": \"2\"")
+        .append(" }]");
+    logger.log(Level.INFO, "RestartVersion Patch String: {0}", patchStr);
+
+    patch = new V1Patch(new String(patchStr));
+    boolean rvPatched = assertDoesNotThrow(() ->
+            patchDomainCustomResource(domainUid, domainNamespace, patch, "application/json-patch+json"),
+        "patchDomainCustomResource(restartVersion)  failed ");
+    assertTrue(rvPatched, "patchDomainCustomResource(restartVersion) failed");
+    logger.info("Snooze for 2 minutes for Introspector to kick off");
+    try {
+      TimeUnit.MINUTES.sleep(2);
+    } catch (java.lang.InterruptedException ie) {
+      logger.info("Got InterruptedException during Thread.sleep");
+      fail("Got InterruptedException during Thread.sleep");
+    }
+    // Verify a rolling restart is triggered in a sequential fashion 
+    // admin-server --> managed-server1 --> managed-server2 
+    logger.info("Check admin server pod {0} to be restarted in ns {1}",
+        adminServerPodName, domainNamespace);
+    checkPodCreated(adminServerPodName, domainUid, domainNamespace);
+    checkPodReady(adminServerPodName, domainUid, domainNamespace);
+    checkServiceCreated(adminServerPodName, domainNamespace);
+    checkServerReadyStatusByExec(adminServerPodName, domainNamespace);
+
+    // check managed server pods re-started sequentially
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server pod {0} to be restarted in ns {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkPodCreated(managedServerPrefix + i, domainUid, domainNamespace);
+      checkPodReady(managedServerPrefix + i, domainUid, domainNamespace);
+      checkServiceCreated(managedServerPrefix + i, domainNamespace);
+    }
+
+    String newAdminPodCreationTime = getPodCreationTimestamp(domainNamespace, "", adminServerPodName);
+    assertNotNull(newAdminPodCreationTime, "adminPodCreationTime returns NULL");
+    logger.info("NewAdminPodCreationTime {0} ", newAdminPodCreationTime);
+    if (Long.parseLong(newAdminPodCreationTime) == Long.parseLong(adminPodCreationTime)) {
+      logger.info("NewAdminPodCreationTime {0} BeforeAdminPodCreationTime ${1}",
+          newAdminPodCreationTime, adminPodCreationTime);
+      fail("New pod creation time must be later than the original timestamp");
+    }
```

Comment:
We may have to add a new assertion methods to verify the creation time stamp is different for all pods which are restarted.  Lets create a JIRA for that task.

Reply:
Yes, it is a good idea to have a new Assertion to check the podCreationTimestamp is unique when the domain is (rolling) restarted.